server/game/gamesteps/triggeredabilitywindow.js

```diff
@@ -35,8 +35,16 @@ class TriggeredAbilityWindow extends BaseAbilityWindow {
         return false;
     }
 
+    isCancellableEvent(player) {
+        let cancellableEvents = ['onCardAbilityInitiated', 'onClaimApplied'];
+
+        return !player.noTimer && this.abilityType === 'cancelinterrupt' && _.any(this.events, event => {
```

Comment:
'cancelinterrupt' is not quite right if we care about hiding information for Vengeance for Elia. Hand's Judgment and Treachery are in onCardAbilityInitiated:cancelinterrupt, but Vengeance is in onClaimApplied:interrupt.